docs/documentation/server_admin/topics/sessions/offline.adoc

```diff
@@ -22,12 +22,11 @@ To issue an offline token, users must have the role mapping for the realm-level
 
 Clients can request an offline token by adding the parameter `scope=offline_access` when sending their authorization request to {project_name}. The {project_name} OIDC client adapter automatically adds this parameter when you use it to access your application's secured URL (such as, $$http://localhost:8080/customer-portal/secured?scope=offline_access$$). The Direct Access Grant and Service Accounts support offline tokens if you include `scope=offline_access` in the authentication request body.
 
-Offline sessions are besides the Infinispan caches stored also in the database. Whenever the {project_name} server is restarted or an offline session is evicted from the Infinispan cache, it is still available in the database. Any following attempt to access the offline session will load the session from the database, and also import it to the Infinispan cache.
+{project_name} will limit its internal cache for offline user and offline client sessions to 10000 entries by default, which will reduce the overall memory usage for offline sessions.
```

Comment:
isn't true for "online" sessions too?

Reply:
Yes, it is true also for "online" sessions. Still this whole chapter is about offline sessions, so I'd think it is ok to limit it to the offline session scope here.